app/javascript/articles/components/Meta.jsx

```diff
@@ -0,0 +1,87 @@
+import { h } from 'preact';
+import {
+  articlePropTypes,
+  organizationPropType,
+} from '../../src/components/common-prop-types';
+import { PublishDate } from './PublishDate';
+
+export const Meta = ({ article, organization }) => (
+  <div className="crayons-story__meta">
+    <div className="crayons-story__author-pic">
+      {organization && !document.getElementById('organization-article-index') && (
+        <a
+          href={`/${organization.slug}`}
+          className="crayons-logo crayons-logo--l"
+        >
+          <img
+            alt={`${organization.name} logo`}
+            src={organization.profile_image_90}
+            className="crayons-logo__image"
+            loading="lazy"
+          />
+        </a>
+      )}
+      <a
+        href={`/${article.user.username}`}
+        className={`crayons-avatar ${
+          organization && !document.getElementById('organization-article-index')
+            ? 'crayons-avatar--s absolute -right-2 -bottom-2 border-solid border-2 border-base-inverted'
+            : 'crayons-avatar--l'
+        }`}
+      >
+        <img
+          src={article.user.profile_image_90}
+          alt={`${article.user.username} profile`}
+          className="crayons-avatar__image"
+          loading="lazy"
+        />
+      </a>
+    </div>
+    <div>
+      <p>
+        <a
+          href={`/${article.user.username}`}
+          className="crayons-story__secondary fw-medium"
+        >
+          {filterXSS(
+            article.class_name === 'User'
+              ? article.user.username
+              : article.user.name,
+          )}
+        </a>
+        {organization &&
+          !document.getElementById('organization-article-index') && (
```

Comment:
Also, if this check is done more than once, consider just popping it into a boolean variable.

Reply:
I'm not clear about the check, but I've extracted it to a variable!